docker/driver-loader/docker-entrypoint.sh

```diff
@@ -136,3 +136,5 @@ else
 fi
 
 /usr/bin/falcoctl driver install --compile=$ENABLE_COMPILE --download=$ENABLE_DOWNLOAD --http-insecure=$HTTP_INSECURE --http-headers="$FALCOCTL_DRIVER_HTTP_HEADERS" $extra_args
+
+exec /usr/bin/falco
```

Comment:
Why can't we use `CMD` in the docker file? :thinking:

Reply:
How is it supposed to work? Unless `exec "$@"` as the entrypoint latest command; is this what you are referring to?

Reply:
Done (using `exec "$@"` in the entrypoint).